src/pymor/models/basic.py

```diff
@@ -188,7 +185,10 @@ def with_time_stepper(self, **kwargs):
     def _compute_solution(self, mu=None, **kwargs):
         mu = mu.with_(t=0.)
         U0 = self.initial_data.as_range_array(mu)
-        U = self.time_stepper.solve(operator=self.operator, rhs=self.rhs, initial_data=U0, mass=self.mass,
+        U = self.time_stepper.solve(operator=self.operator,
+                                    rhs=None if isinstance(self.rhs, ZeroOperator) else self.rhs,
+                                    initial_data=U0,
+                                    mass=None if isinstance(self.mass, IdentityOperator) else self.mass,
```

Comment:
Would it not be nicer to not have the check here and instead extend the check/switch in the timestepper? It already checks for `None` and could also check for `IdentityOperator` or `ZeroOperator`, respectively.

Reply:
Yeah, maybe, but I didn't want to touch the time steppers in this pr.